doc/build-unix.md

```diff
@@ -44,7 +44,7 @@ Finally, clang (often less resource hungry) can be used instead of gcc, which is
 
 Build requirements:
 
-    sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
+    sudo apt-get install build-essential cmake pkg-config bsdmainutils python3
 
 Now, you can either build from self-compiled [depends](#dependencies) or install the required dependencies:
```

Comment:
Is it worth quickly mentioning for those who are not familiar with depends that depends is for static linking?

Reply:
I'm not oppose to this suggestion, but this PR is focused on rather switching build systems than overall improving.